### PR TITLE
Remove unused LoadJob v1 code

### PR DESCRIPTION
**Pull request by meegoo**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [ ] enhancement
- [x] others

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

Refactor class source since it will be core concept for stream processing

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 7 / 7 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/load/PartitionLoadInfo.java|4|4|100.00%|
|:large_blue_circle:|com/starrocks/load/BucketLoadInfo.java|3|3|100.00%|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 30 / 33 (90.91%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/load/BucketLoadInfo.java|16|18|88.89%|
|:large_blue_circle:|com/starrocks/load/PartitionLoadInfo.java|14|15|93.33%|

File: fe/fe-core/src/main/java/com/starrocks/load/BucketLoadInfo.java [hunk 1]
```diff
@@ -37,7 +37,7 @@
 import java.util.Map;
 import java.util.Map.Entry;
 
-public class Source implements Writable {
+public class BucketLoadInfo implements Writable {
```
Comment (imay) [inline fe/fe-core/src/main/java/com/starrocks/load/BucketLoadInfo.java:40]: Why name it with bucket? I think bulk or batch is more suitable?

**Review by imay — DISMISSED** [review]
LGTM

**Comment by meegoo** [comment]
run starrocks_fe_unittest

**Comment by meegoo** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:disappointed: **fail** : 1 / 2 (50.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/load/loadv2/LoadJob.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/load/loadv2/JobState.java|1|1|100.00%|

File: fe/fe-core/src/main/java/com/starrocks/load/loadv2/JobState.java [hunk 2]
```diff
@@ -29,5 +29,6 @@
     LOADING, // job is running
     COMMITTED, // transaction is committed but not visible
     FINISHED, // transaction is visible and job is finished
+    QUORUM_FINISHED, // compatible with load v1
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/load/loadv2/JobState.java:32]: `QUORUM_FINISHED` has been not used and i think it can be removed.

**Comment by wyb** [comment]
Why remove `BrokerLoadJobTest.java`?

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:disappointed: **fail** : 0 / 1 (00.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/load/loadv2/LoadJob.java|0|1|00.00%|

**Review by wyb — APPROVED** [review]

**Review by imay — APPROVED** [review]

**Pull request merged**